src/api/index.js

```diff
@@ -62,6 +65,9 @@ RippleAPI.prototype = {
   prepareTrustline,
   prepareOrder,
   prepareOrderCancellation,
+  prepareSuspendedPaymentCreate: prepareSusPayCreate,
+  prepareSuspendedPaymentFinish: prepareSusPayFinish,
+  prepareSuspendedPaymentCancellation: prepareSusPayCancel,
```

Comment:
@clark800 will almost certainly want you to `const prepareSuspendedPaymentCreate = require('./transaction/suspay-create');` to keep his pretty export list.